crates/tessera-storage/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+pub mod shield;
+pub mod storages;
+
+#[trait_variant::make(Storage: Send + Sync)]
+pub trait LocalStorage {
+    type Key: ToOwned + ?Sized + Sync;
+    type Value: ToOwned + ?Sized + Sync;
+    type StorageError: std::error::Error;
+    async fn get(&self, key: &Self::Key) -> Result<<Self::Value as ToOwned>::Owned, Self::StorageError>;
+    async fn set(&self, key: &Self::Key, value: &Self::Value) -> Result<(), Self::StorageError>;
+    async fn delete(&self, key: &Self::Key) -> Result<(), Self::StorageError>;
+    async fn list(
+        &self,
+        key: &Self::Key,
```

Comment:
The parameter name 'key' seems a bit unclear. It would be helpful to explicitly indicate that it’s a parameter used as a path prefix.

Reply:
https://github.com/CremitHQ/tessera/pull/20/commits/deb939412dc8ae2f09ba461d228939444cf48c56